data/spelling-dict/hunspell/pt_PT_45.dic

```diff
@@ -60040,4 +60040,12 @@ Sears
 spinning
 Bronx
 Queens
-Wyoming
\ No newline at end of file
+Wyoming
+sub�ptimo/fp
+z�-ningu�m
+z�s-ningu�m
```

Comment:
Hyphenated words should go in the compound dictionaries.

Reply:
Ahhh... sorry... give me a few minutes, I will remove them from there.

Reply:
Done! Removed them from the .dics. Will add to compound dictionaries some other time.